tests/models/test_omnivore.py

```diff
@@ -41,19 +41,19 @@ def test_omnivore_swin_t_forward(omnivore_swin_t_model, device):
 
     assert_expected(image_score.size(), torch.Size((1, 1000)))
     assert_expected(
-        image_score.abs().sum(), torch.tensor(200.27574), rtol=1e-3, atol=1e-3
+        image_score.abs().sum(), torch.tensor(184.01417), rtol=1e-3, atol=1e-3
```

Comment:
Since the underlying SwinTransformer3d implementation changed (probably changed on some ordering or other thing but the logic stay the same), the random initiation of the weight also change, hence we need to update the expected value here.